Source/Extensions/Blazorise.Components/Autocomplete.razor.cs

```diff
@@ -916,16 +928,15 @@ public TValue SelectedValue
         [Parameter] public bool CloseOnSelection { get; set; } = true;
 
         /// <summary>
-        /// Shows selected option(s) when presenting the searched values.
+        /// Suggests already selected option(s) when presenting the options.
         /// </summary>
-        [Parameter] public bool ShowSelected { get; set; }
+        [Parameter] public bool SuggestSelectedItems { get; set; }
 
         /// <summary>
-        /// Shows selected option(s) when presenting the searched values with checkboxes to ease selection when <see cref="Autocomplete{TItem, TValue}"/> is set to Multiple.
+        /// Suggests already selected option(s) when presenting the options with checkboxes to ease selection when <see cref="Autocomplete{TItem, TValue}"/> is set to Multiple.
         /// </summary>
         [Parameter] public bool ShowMultipleCheckbox { get; set; }
```

Comment:
Should this also be renamed to Suggest prefix?

Reply:
Or just `MultipleCheckbox`? I'm not sure the Suggest really applies here. What do you prefer?

Reply:
Well, I think checkboxes should always be visible so that they can be checked and unchecked. Not sure what is the behavior for ShowMultipleCheckbox ?